src/coreclr/debug/di/eventredirectionpipeline.cpp

```diff
@@ -108,9 +108,10 @@ HRESULT EventRedirectionPipeline::AttachDebuggerToTarget(LPCWSTR szOptions, DWOR
         // Initialize
         m_pBlock->m_versionCookie = EVENT_REDIRECTION_CURRENT_VERSION;
 
-        s.Printf(m_CommonParams, GetCurrentProcessId(), m_pBlock, szOptions, pidTarget);
-        lpCommandLine = s.GetUnicode();
+        int res = swprintf_s(buffer, m_CommonParams, GetCurrentProcessId(), m_pBlock, szOptions, pidTarget);
```

Comment:
Is this Windows specific, or just delaying the problem with `swprintf_s` till later?

Reply:
This is windows specific. This code path is specifically for the strikeRS feature that was never ported to CoreCLR.

Reply:
Should this be deleted instead?

Reply:
@noahfalk Should this unused code be deleted?

Reply:
IMO, yeah we could delete this stuff. I am not aware of it getting any use for many years. @dotnet/dotnet-diag - Does anyone else have concerns about concerns about us deleting this?

@AaronRobinsonMSFT - I'd say give folks 24 hours to respond and if we don't have any objections consider that the green light to delete this code if you'd like.

Reply:
strikeRS is something @tommcdon was looking at, but I was of the same opinion. This is essentially old code that's just adding maintenance burden right now.

Reply:
I've created a separate commit that removes it and all uses for the redirection infrastructure. Please let me know if I should revert the commit.

Reply:
No concerns from my end removing strikers support.  We can always revisit it if we decide to enable support in the future.

Reply:
Thanks @tommcdon and @hoyosjs. Given your responses coupled with @noahfalk I am going to assume this is enough representation from the Diagnostics team that I can merge this in when green. Appreciate the quick responses.